### PR TITLE
New: PointTool to draw points

### PR DESCRIPTION
**Pull request by arnauddhaene**
* Use a circle that cannot be stretched using onMouseMove
* Modify the x and y parameters so that a mouse click will place center

### CODE REVIEW

**Comment by arnauddhaene** [comment]
Hi @andfanilo,

Here is the implementation I mentioned in my comment on [this issue](https://github.com/andfanilo/streamlit-drawable-canvas/issues/68#issuecomment-1050521220). I don't think I have the rights to link the issue to this PR directly, feel free to handle that on your side.

Let me know what you think!

File: streamlit_drawable_canvas/frontend/src/lib/point.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  /**
+   * Calculate the distance of two x,y points
+   *
+   * @param point1 an object with x,y attributes representing the start point
+   * @param point2 an object with x,y attributes representing the end point
+   *
+   * @returns {number}
+   */
+  linearDistance = (point1: any, point2: any) => {
```
Comment (andfanilo) [inline streamlit_drawable_canvas/frontend/src/lib/point.ts:87]: I'm pretty sure you can remove this section, it was only used to compute the radius of the circle in realtime but you have it fixed

Comment (arnauddhaene) [reply streamlit_drawable_canvas/frontend/src/lib/point.ts]: Good catch. Just pushed a fix where I removed that part.

**Comment by andfanilo** [comment]
* Good thinking about editing the `README.md`. You'll also need to edit the docstring in `__init__.py` to add the `point` choice.
* If you feel comfortable enough with doing this, is there a way to put the radius of the circle as argument of the Python call and have it propagated to the radius of the point? Otherwise you can create an issue for it and we can go back to this afterwards

**Comment by arnauddhaene** [comment]
Hey @andfanilo, I was able to implement the additional radius parameter on `st_canvas` and complete the downstream pipeline. I added a slider to test it out in the `app.py` example present in the`README.md`. Let me know what you think!

**Comment by andfanilo** [comment]
This LGTM! Let me merge and play with it too :)

**Pull request merged**